# Source: Internal
# Info: Written internally- removing -Wall flags because Solaris Studio does not know of them  
# Status: This patch is Solaris specific.  Will not be sent upstream.

--- ORIGINAL/src/Makefile.am	2016-01-12 08:43:52.928382761 -0800
+++ gparted-0.25.0/src/Makefile.am	2016-01-12 08:46:21.389297831 -0800
@@ -3,10 +3,6 @@
 	-DGPARTED_DATADIR=\""$(datadir)"\"			\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\"
 
-AM_CFLAGS = -Wall	
-
-AM_CXXFLAGS = -Wall	
-
 sbin_PROGRAMS = gpartedbin
 
 gpartedbin_SOURCES = \
